src/common/uri.ts

```diff
@@ -43,9 +45,34 @@ export interface GitUriOptions {
 	base: boolean;
 }
 
-// As a mitigation for extensions like ESLint showing warnings and errors
-// for git URIs, let's change the file extension of these uris to .git,
-// when `replaceFileExtension` is true.
+const ImageMimetypes = [
+	'image/png',
+	'image/gif',
+	'image/jpeg',
+	'image/webp',
+	'image/tiff',
+	'image/bmp'
+];
+
+export async function asImageDataURI(uri: Uri, repository: Repository): Promise<Uri | undefined> {
+	try {
+		const { commit } = JSON.parse(uri.query);
```

Comment:
for `pr` scheme uris, the query has `headCommit`, `baseCommit`, `isBase`. We could try to use those or just short-circuit for non-review scheme uris

Reply:
Changed this and the call up a bit, now we can view added, modified, and deleted images in both `pr` and `review` schemes.